todo_app/templates/layout.html

```diff
@@ -10,11 +10,14 @@
   </head>
   <body>
     <nav class="navbar navbar-expand-md navbar-dark bg-dark mb-4">
-      <a class="navbar-brand" href="/">To-Do App</a>
+      <img src="green-tick.png" width="30" height="30" alt="">
```

Comment:
By default, Flask looks for images in a folder called `static` next to your templates, i.e. folder structure like:
```
todo_app
-> templates
--> index.html
--> etc.
-> static
--> green-tick.png
```
And then you can access that with `src="static/green-tick.png"`

If you want to customise that behaviour there's [a good discussion here on Stack Overflow](https://stackoverflow.com/questions/20646822/how-to-serve-static-files-in-flask)